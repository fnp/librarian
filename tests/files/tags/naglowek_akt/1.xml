<naglowek_akt>AKT PIERWSZY</naglowek_akt>

Pokój w zamku Cześnika, drzwi na prawo, lewo i w środku, stoły, krzesła etc., gitara angielska na ścianie.
<naglowek_scena>SCENA PIERWSZA</naglowek_scena>
