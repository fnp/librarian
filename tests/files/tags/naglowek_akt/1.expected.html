<div id="book-text">
<div id="toc">
<h2>Spis tre≈õci</h2>
<ol>
<a href="#s1" class="anchor">AKT PIERWSZY</a><li><ol>
<a href="#s2" class="anchor">SCENA PIERWSZA</a><li>
</ol></li>
</ol>
</div>
<h2 class="wl" id="s1">AKT PIERWSZY</h2>
<h3 id="s2">SCENA PIERWSZA</h3>
</div>
